methods.lua

```diff
@@ -631,6 +631,34 @@ function api.sendVideo(chat_id, video, duration, caption, reply_to_message_id)
 
 end
 
+
+function api.sendVideoNote(chat_id, video_note, duration, reply_to_message_id)
+	
+	local url = BASE_URL .. '/sendVideoNote'
```

Comment:
Why the random indents?
I'm guessing it's your style, but it's probably best if you use the style of the rest of the code when contributing :)